Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ however, it has to be formatted properly to pass verification tests.
 ## [Unreleased] - YYYY-MM-DD
 
 ### Fixed
+- Default UI actions would not function without a user specifically setting up project-wide actions in Project Settings [ISXB-811](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-811).
```

Comment:
I would recommend rewriting this in the format that explains the problem+symptom and new behavior. E.g. (not sure if below is 100% correct, but just to illustrate what I mean)

Fixes an issue where UGUI or UI Toolkit would not react to input unless a Project-wide Input Action Asset was configured for the project. If no Project-wide Input Action Asset has been configured for the project, default actions are now instead created on-the-fly to provide default bindings for UI actions.

Reply:
Tweaked this in 982b8abdd43d4e7f05fa560d305f2f5f972c6fef.